tests/unit/expire.tcl

```diff
@@ -163,8 +163,8 @@ start_server {tags {"expire" "external-ok"}} {
         set size1 [r dbsize]
         # Redis expires random keys ten times every second so we are
         # fairly sure that all the three keys should be evicted after
-        # one second.
-        after 1000
+        # two seconds.
+        after 2000
```

Comment:
1. i think we should use a wait-for-condition here with a timeout rather than an sleep. (that's actually true for all sleeps).
2. any test that's time-dependent (has a long sleep, or a wait_for that waits for something in redis that takes a long time), should have a `slow` tag, so that we can run the test suite skipping inherently slow tests
3. i'd like the slow skip tag to somehow be used by default? (i want it to be an opt-in, like `--accurate` rather than opt-out), or at the very least, i want to skip these in the CI.yml runs, since we keep adding more and more of these.

@sundb FYI (in the context of #9003)